index.js

```diff
@@ -70,8 +87,6 @@ HyperdriveSwarm.prototype._node = function () {
   })
 
   swarm.listen(self.opts.port || 3282)
-  self.node = swarm
+  self.discoverySwarm = swarm
```

Comment:
i dont think its necessary to change the API here. let's stay as backwards compatible as possible